dotcom-rendering/src/web/components/Toast.tsx

```diff
@@ -1,27 +1,33 @@
 import { css } from '@emotion/react';
 import { EditorialButton } from '@guardian/source-react-components-development-kitchen';
 import { Hide, SvgArrowUpStraight } from '@guardian/source-react-components';
+import { between, space } from '@guardian/source-foundations';
 import { getZIndex } from '../lib/getZIndex';
 
 type Props = {
 	count: number;
 	onClick: () => void;
 	format: ArticleFormat;
+	stuck: boolean;
 };
 
 /**
  * A button to scroll to the top when new content exists.
  */
-export const Toast = ({ count, onClick, format }: Props) => {
+export const Toast = ({ count, onClick, format, stuck }: Props) => {
 	return (
 		<div
 			css={css`
-				position: fixed;
-				top: 20px;
-				display: flex;
-				${getZIndex('toast')};
+				position: ${stuck ? 'fixed' : 'absolute'};
```

Comment:
I’m assuming we’ve tested the css sticky, here?

Reply:
Yeah. But I can't remember why it didn't work... and now when I try it again it works flawlessly.

I think I'm going to just pretend I went with sticky all along and that none of this ever happened.